test/logger_test.rb

```diff
@@ -455,9 +455,9 @@ class TestLogger < Hanami::Logger; end
           output =
             stub_stdout_constant do
               class TestLogger < Hanami::Logger; end
-              TestLogger.new.info('This is a message with spaces')
+              TestLogger.new.info("This is a message with quotes \" ' and with spaces")
```

Comment:
This is cheating :wink: 

Shall we try with `TestLogger.new.info(%(This is a message with quotes " ' and with spaces))` instead? I'm afraid that we're going down the rabbit hole. We should find a way to escape strings.